internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -0,0 +1,349 @@
+package advancedcluster
+
+import (
+	"context"
+
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/conversion"
+)
+
+type TfLabelModel struct {
```

Comment:
just out of convention from other resource would keep terraform related models in same file were schema is defined.

Reply:
so the idea of model_*.go files is just to have conversion methods?

Reply:
yes, and model_*_test.go for testing those conversion methods. I imagine we will likely have schema and terraform model definitions also in a separate file when we starting autogenerating them from the api spec.